src/DataTypes/PaymentMethod.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace CommerceGuys\AuthNet\DataTypes;
+
+interface PaymentMethod
```

Comment:
No problem. I've pushed that change. I should've caught that by looking at the other code. :-)